extensions/rich_text_components/Math/Math.py

```diff
@@ -32,6 +32,7 @@ class Math(base.BaseRichTextComponent):
         'schema': {
             'type': 'custom',
             'obj_type': 'MathLatexString',
+            'placeholder_text': '\\frac{x}{y}'
```

Comment:
Based on what you suggested I am running into a problem in math_latex_string_editor.html.

I am not able to figure out how do I modify this line:
`<span ng-if="!localValue.label" class="placeholder-text" mathjax-bind="placeholderText"></span>`.
This line does not work:
`<span ng-if="!localValue.label" class="placeholder-text" mathjax-bind="\frac{x}{y}"></span>`. 
How do I best modify this line to make it work?